### PR TITLE
Fix keycheck stuff, Use API instead, Remove sepolicy, sed -e is only for multiple patches

### PR DESCRIPTION
**Pull request by JohnFawkes**
Signed-off-by: JohnFawkes <33303104+JohnFawkes@users.noreply.github.com>

### CODE REVIEW

**Comment by lzlrd** [comment]
@JohnFawkes You didn't remove sepolicy in that commit?

**Comment by lzlrd** [comment]
@Stallix do they have to use beta Dialer for Call Screening?

**Comment by ThatNerdyPikachu** [comment]
@lazerl0rd Yes. (at least, from what we've all tested, you need the latest public beta of the Dialer (``27.0.225573798-publicbeta``)) for Call Screening to work.

File: common/install.sh [hunk 1]
```diff
@@ -80,18 +81,18 @@ chooseold() {
...
-$FUNCTION "DOWN"
-# fi
+if keytest; then
+  FUNCTION=choose
+else
+  FUNCTION=chooseold
+  ui_print "   ! Legacy device detected! Using old keycheck method"
+  ui_print " "
+  ui_print " - Vol Key Programming -"
+  ui_print "   Press Vol Up Again:"
```
Comment (ThatNerdyPikachu) [inline common/install.sh:91]: Why does this have to say again, or am I just missing something?

File: common/install.sh [hunk 2]
```diff
@@ -108,21 +109,18 @@ if $FUNCTION; then
...
-  sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
-  rm -rf $INSTALLER/system/vendor/overlay/Pixel
+  sed -i 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
   rm -f $INSTALLER/system/vendor/overlay/Pix3lify.apk
-  rm -rf /data/resource-cache
+  rm -rf $INSTALLER/system/vendor/overlay/Pixel
   rm -rf /data/dalvik-cache
+  rm -rf /data/resource-cache
   ui_print "   Dalvik-Cache has been cleared!"
   ui_print "   Next boot may take a little longer to boot!"
```
Comment (ThatNerdyPikachu) [inline common/install.sh:125]: Can we say something like ``Next boot may take a little longer``, but not because that sounds.. off?

**Review by ThatNerdyPikachu — APPROVED** [review]

**Pull request merged**